src/parsing.rs

```diff
@@ -276,8 +276,8 @@ pub fn get_syntax(media_type: MediaType) -> Syntax {
       allow_return_outside_function: true,
       allow_super_outside_method: true,
       auto_accessors: true,
-      decorators: false,
-      decorators_before_export: false,
+      decorators: true,
+      decorators_before_export: true,
```

Comment:
Two things:
- I'm not sure if this should be enabled by default, but adding a new argument to `get_syntax` is gonna be painful
- `decorators_before_exports` needs to be verified - I think it's a Babel transform that we don't necessarily want to enable

Reply:
I did search through TC39 proposal repo but couldn't find an obvious answer. @nicolo-ribaudo any chance you could chime in here and suggest the proper default?

Reply:
The proposal now settled to decorators _after_ export, so this should default to `false` (and ideally not be configurable at all by users). We are going to remove this option in the next Babel major btw.

Reply:
Lovely, thanks!